Find a functionally equivalent code:android.graphics.Canvas.translate:COMMENT
<android.graphics.Canvas: void translate(float,float)>
Method Modifier: public      native      
Method Modifier: Yes         Yes         
Method Updated:  Yes         No          
Comment:/**
 * Preconcat the current matrix with the specified translation
 *
 * @param dx The distance to translate in X
 * @param dy The distance to translate in Y
 */

@@ -1 +1,3 @@
-
+{
+    native_translate(mNativeCanvasWrapper, dx, dy);
+}

